src/modules/form/components/group/FormCard.tsx

```diff
@@ -162,7 +183,28 @@ const FormCard: React.FC<Props> = ({
           }}
         >
           {renderChildItem &&
-            item.item?.map((childItem) => renderChildItem(childItem))}
```

Comment:
Rather than wrapping renderChildItem in an additional Box in FormCard, I wonder if it would be easier and more flexible to add another variant to the form definition (say 'ACCENT' or 'HIGHLIGHT') for these inner group components. Then QuestionGroup could handle rendering the appropriate border, much as it does for `isConditionalGroup`. I think that gets us out of passing around groupHeaderProps around too. It could also allow you to have children of a SIGNATURE group that don't have this accent treatment.

Happy to chat about this approach is it would be helpful.

Reply:
Good plan! Thank you. I think the component is actually not a `QuestionGroup` but a `HorizontalGroup` so I added it there, but let me know if you think I should add handling of these variant styles to the `QuestionGroup` as well?